### DISCUSSION TITLE
Eager loading with Pydantic model and SQLAlchemy?

### DISCUSSION THREAD

**Discussion by adityar15** [fastapi/fastapi#8447] [Questions] [answered]
### First Check

- [x] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
#sqlalchemy models

class Seller(db.Base):
    __tablename__= 'sellers'
    id = Column(Integer, primary_key=True, index=True)
    name = Column(String(100))
    products = relationship("Product", backref="seller")


class Product(db.Base):
    __tablename__= 'products'
    id = Column(Integer, primary_key=True, index=True)
    seller_id = Column(Integer, ForeignKey('sellers.id'))
    name = Column(String(100))
    attributes = relationship("Attribute", backref="product")


class Attribute(db.Base):
    __tablename__ = 'attributes'
    id = Column(Integer, primary_key=True, index=True)
    product_id =  Column(Integer, ForeignKey('products.id'))
    name = Column(String(255), unique=True, index=True, nullable=False)

```

### Description

I am trying to eager load models in Fast API. Based on the example above I want to eager load seller data and go down the tree till attribute. How to eager load it? 

### Operating System

Windows

### FastAPI Version

0.65.1

### Python Version

3.8.7

**Comment by jgould22** [comment, marked as answer]
This is more of an SQLA question than a fastapi question.

Regardless all the various techniques and their associated trade offs (and how to use them) are listed in the SQLA docs here https://docs.sqlalchemy.org/en/14/orm/loading_relationships.html
